README.md

```diff
@@ -116,7 +116,7 @@ learning and docs
 
 raylib is designed to be learned using [the examples](https://github.com/raysan5/raylib/tree/master/examples) as the main reference. There is no standard API documentation but there is a [**cheatsheet**](https://www.raylib.com/cheatsheet/cheatsheet.html) containing all the functions available on the library a short description of each one of them, input parameters and result value names should be intuitive enough to understand how each function works. 
 
-Some additional documentation about raylib design can be found in the raylib GitHub Wiki. Here are the relevant links:
+Some additional documentation about raylib design can be found in   raylib GitHub Wiki. Here are the relevant links:
```

Comment:
It seems there are some extra spaces in the sentence. Please, could you review it.